image/models.py

```diff
@@ -14,6 +14,16 @@
 import boto3
 import wevote_functions.admin
 
+# naming convention stored at aws
+FACEBOOK_PROFILE_IMAGE_NAME = "facebook_profile_image"
+FACEBOOK_BACKGROUND_IMAGE_NAME = "facebook_background_image"
+TWITTER_PROFILE_IMAGE_NAME = "twitter_profile_image"
+TWITTER_BACKGROUND_IMAGE_NAME = "twitter_background_image"
+TWITTER_BANNER_IMAGE_NAME = "twitter_banner_image"
+MAPLIGHT_IMAGE_NAME = "maplight_image"
+VOTE_SMART_IMAGE_NAME = "vote_smart_image"
+MASTER_IMAGE = "master"
+
```

Comment:
Please make the database text match the python constant. So "facebook_profile_image" should be "FACEBOOK_PROFILE_IMAGE_NAME".